research/src/fetch_campaign_donations.py

```diff
@@ -0,0 +1,306 @@
+                    'Município': 'city',
+                    'Nome candidato': 'candidate_name',
+                    'Nome da UE': 'electoral_unit_name',
+                    'Nome do doador': 'donor_name',
+                    'Nome do doador (Receita Federal)':
+                    'donor_name_for_federal_revenue',
+                    'Nome do doador originário': 'original_donor_name',
+                    'Nome do doador originário (Receita Federal)':
+                    'original_donor_name_for_federal_revenue',
+                    'Número candidato': 'candidate_number',
+                    'Número candidato doador': 'donor_candidate_number',
+                    'Número do documento': 'document_number',
+                    'Número partido doador': 'donor_party_number',
+                    'Número Recibo Eleitoral': 'electoral_receipt_number',
+                    'Número UE': 'electoral_unit_number',
+                    'Sequencial Candidato': 'candidate_sequence',
+                    'Sequencial prestador conta': 'accountable_sequence',
+                    'Sequencial comite': 'committee_sequence',
+                    'Setor econômico do doador': 'donor_economic_sector',
+                    'Setor econômico do doador originário':
+                    'original_donor_economic_sector',
+                    'Sigla da UE': 'electoral_unit_abbreviation',
+                    'Sigla Partido': 'party_acronym',
+                    'Sigla UE doador': 'donor_electoral_unit_abbreviation',
+                    'Tipo de documento': 'document_type',
+                    'Tipo doador originário': 'original_donor_type',
+                    'Tipo partido': 'party_type',
+                    'Tipo receita': 'revenue_type',
+                    'Tipo comite': 'committee_type',
+                    'UF': 'state',
+                    'Valor receita': 'revenue_value'}
+    if 'candidates' in donations_data.keys():
+        donations_data['candidates'].rename(columns={translations},
+                                            inplace=True)
+    if 'parties' in donations_data.keys():
+        donations_data['parties'].rename(columns={translations},
+                                         inplace=True)
+    if 'committees' in donations_data.keys():
+        donations_data['committees'].rename(columns={translations},
+                                            inplace=True)
+
+
+def download_base(url, folder_walk_function):
+    file_name = url.split('/')[-1]
+    print("Downloading " + file_name)
+    urllib.request.urlretrieve(url, file_name, reporthook)
+    print("Uncompressing downloaded data.")
+    with zipfile.ZipFile(file_name, "r") as zip_ref:
+        zip_ref.extractall(file_name.split('.')[0])
+    os.remove(file_name)
+
+    print("Reading all the data and creating the dataframes...")
+
+    donations_data = folder_walk_function()
+
+    if 'candidates' in donations_data.keys():
```

Comment:
Here we have 3 `if` statements that change the keyword ('candidates', 'parties' and 'committees') but have almost the same code. Can we do something similar, and use the key as a parameter?